projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.html

```diff
@@ -8,7 +8,11 @@
     [attr.aria-controls]="_panelId"
     (click)="toggle()"
   >
+    <lg-icon *ngIf="!!iconName" [name]="iconName" class="lg-accordion__heading-icon"></lg-icon>
     <ng-content></ng-content>
-    <lg-icon class="lg-accordion__heading-icon" name="chevron-down"></lg-icon>
+    <lg-icon
+       [ngClass]="{'lg-accordion__heading-icon': true, 'lg-accordion__heading-icon--animated': true, 'lg-accordion__heading-icon--right': !!iconName}"
```

Comment:
There are 2 classes here that will be applied by default because they are set to true by default. I suggest just adding them to the `class` attribute instead.

Reply:
I've fixed this, thanks!